department-of-veterans-affairs/vets-website
# Fixes Session Timeout Modal accessibility bug

## Description
Issue: [SessionTimeoutModal language is not WCAG compliant](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16301)
The Session Timeout Modal (SessionTimeoutModal.jsx) is displayed when a user's session will expire (in 30 seconds) and the content is not WCAG compliant (it refers to a button's color for a user action, rather than allowing the content to instruct the user) which is an accessibility issue.

## Testing done
- Manually

## Steps to Test
With a logged in user
1.) In Chrome DevTools > Application tab > Storage (in sidebar) > Local Storage >
2.) Right-click on `sessionExpiration` or `sessionExpirationSSO` and select 'Edit "Key"'
3.) Modify the time (within in a few minutes) to display the Session Timeout Modal
4.) Verify Session Timeout Modal content is changed.

## Screenshots
[
![Screen Shot 2020-12-01 at 2 50 45 PM](https://user-images.githubusercontent.com/67602137/100794258-4a312d00-33eb-11eb-9c70-66d21bf9fc24.jpeg)
](url)

## Acceptance criteria
- [x] Content specialist approves the text change
- [ ] Change is communicated broadly in Slack
- [ ] Change is verified in Staging environment

## Definition of done
